app/assets/stylesheets/pagination.css

```diff
@@ -0,0 +1,87 @@
+/* Obtained from will-paginate gem */
+
```

Comment:
we are using scss or css ?
please be consistent

Reply:
scss, acknowledged! Modified the extension.